### PR TITLE
chore: use stale action

### PR DESCRIPTION
**Pull request by Lightning00Blade**
Issue https://github.com/puppeteer/puppeteer/issues/12178

Note: we still need to disable the app.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          stale-issue-message: >
+            We're marking this issue as unconfirmed because it has not had recent activity and 
+            we weren't able to confirm it yet. It will be closed if no further activity occurs
+            within the next 30 days.
+          close-issue-message: >
+            We are closing this issue. If the issue still persists in the latest version of
+            Puppeteer, please reopen the issue and update the description. We will try our
+            best to accommodate it!
+          stale-issue-label: 'unconfirmed'
+          exempt-issue-labels: 'confirmed,feature'
```
Comment (OrKoN) [inline .github/workflows/stale.yml:29]: is 30 days the default?

Comment (OrKoN) [reply .github/workflows/stale.yml:29]: (for issues)

**Comment by OrKoN** [comment]
> Note: we still need to disable the app.

How is it done?

File: .github/workflows/stale.yml (hunk 1 above)
Comment (Lightning00Blade) [reply .github/workflows/stale.yml:29]: `days-before-stale` - default `60` (Same as we had before)
`days-before-close` - default `7` (We had 30 days before)

**Comment by Lightning00Blade** [comment]
> How is it done?
From what I understand we just need to revoke the permission to the repo. I found how to do it

**Comment by OrKoN** [comment]
@Lightning00Blade have you done it? let's disable the old app first to avoid clashes.

**Comment by Lightning00Blade** [comment]
> @Lightning00Blade have you done it? let's disable the old app first to avoid clashes.

Done

**Review by OrKoN — APPROVED** [review]

**Pull request merged**